src/components/Chart/TimeBars.docs.md

```diff
@@ -991,15 +991,30 @@ age,date,value
       "value": 6000,
       "label": "Schätzung",
       "showValue": false,
-      "position": "bottom"
+      "position": "bottom",
+      textAlignment: 'right',
     },
     {
-      "x": 74,
+      "x": '74',
       "value": 90712,
       "label": "Tatsächlich",
       "showValue": false,
-      "position": "bottom"
-    }
+      "position": "bottom",
+      "textAlignment": 'right'
+
+    },
+    {
+      x1: '0',
+      x2: '70',
+      value: 68139,
+      label: '70. Perzentile',
+      showValue: true,
+      unit: 'Franken',
+      position: 'bottom',
+      ghost: true,
+      textAlignment: 'left',
```

Comment:
Didn't mention it before on the call, but don't you think the `textAlignment`should automatically be `left`when `leftLabel`is `true`? The we can have only `leftLabel` option on here.